content/en/news/2019/announcing-1.4/change-notes/index.md

```diff
@@ -16,6 +16,12 @@ weight: 10
 
 ## Telemetry
 
+- **Added** experimental [in-proxy](/docs/ops/telemetry/in-proxy-service-telemetry/) telemetry reporting to Stackdriver.
```

Comment:
@mandarjog any info on how to enable this?

Reply:
I think we should create a wiki page for it. @mandarjog wdyt?

Reply:
Any updates?

Reply:
I will create a wiki page.

Reply:
Thanks!

Reply:
Here is the wiki page: https://github.com/istio/istio/wiki/Mixerless-Stackdriver-HTTP-Telemetry

Reply:
I hate to be the one to say - but our release notes should really not link to blog posts (they are not subject to versioning and have many other weaknesses.  As you have blog post content, perhaps you can convert to an MD formatted file and submit as a document.

Reply:
Sure, created a readme file: https://github.com/istio/proxy/pull/2545. Let's replace the wiki page with that.